cmd/evm/main.go

```diff
@@ -240,21 +240,20 @@ func (ruleSet) GasTable(*big.Int) *vm.GasTable {
 }
 
 func (self *VMEnv) RuleSet() vm.RuleSet       { return ruleSet{} }
-func (self *VMEnv) Vm() vm.Vm                 { return self.evm }
 func (self *VMEnv) Db() vm.Database           { return self.state }
 func (self *VMEnv) SnapshotDatabase() int     { return self.state.Snapshot() }
 func (self *VMEnv) RevertToSnapshot(snap int) { self.state.RevertToSnapshot(snap) }
 func (self *VMEnv) Origin() common.Address    { return *self.transactor }
-func (self *VMEnv) BlockNumber() *big.Int     { return new(big.Int) }
-func (self *VMEnv) Coinbase() common.Address  { return *self.transactor }
-func (self *VMEnv) Time() *big.Int            { return self.time }
-func (self *VMEnv) Difficulty() *big.Int      { return common.Big1 }
-func (self *VMEnv) BlockHash() []byte         { return make([]byte, 32) }
-func (self *VMEnv) Value() *big.Int           { return self.value }
-func (self *VMEnv) GasLimit() *big.Int        { return big.NewInt(1000000000) }
-func (self *VMEnv) VmType() vm.Type           { return vm.StdVmTy }
-func (self *VMEnv) Depth() int                { return 0 }
-func (self *VMEnv) SetDepth(i int)            { self.depth = i }
+func (self *VMEnv) BlockNumber() *big.Int    { return new(big.Int) }
```

Comment:
Seems like a whitespace typo?

Reply:
Hm, looks like a some sort of IDE autoformating. Is it important? I can format it back of course.

Reply:
Not very important -- `gofmt` can take care of it anytime/before merge